convert_trace_to_json.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Can we either do this in haskell or tidy the script away under `test` somewhere so it's not cluttering up the root directory.